packages/daylight/daylight_start.go

```diff
@@ -324,6 +325,10 @@ func Start(dir string, thrustWindowLoder *window.Window) {
 
 	// database rollback to the specified block
 	if *utils.RollbackToBlockID > 0 {
+		err = syspar.SysUpdate()
+		if err != nil {
+			log.Errorf("can't read system parameters: %s", utils.ErrInfo(err))
```

Comment:
That's incorrect usage of logrus. You should do like that: log.WithFields(log.Fields{"error": err, "type": consts.DBError}).Error("cant read system paramters").